app/globals.css

```
@@ -166,6 +166,20 @@
   animation: fade-in-up 1.8s ease-out;
 }
 
+@utility animate-bounce-gentle {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)